Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/RatioCounterGauge.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.WebAppPerformanceCollector
+{
+    /// <summary>
+    /// Gauge that sums up the values of different gauges.
+    /// </summary>
+    internal class RatioCounterGauge : ICounterValue
+    {
+        /// <summary>
+        /// The numerator gauge used to compute the target ratio.
+        /// </summary>
+        private readonly ICounterValue numeratorGauge;
+
+       /// <summary>
+        /// The denominator gauge used to compute the target ratio.
+        /// </summary>
+        private readonly ICounterValue denominatorGauge;
+
+        /// <summary>
+        /// Name of the counter.
+        /// </summary>
+        private string name;
+
+        /// <summary>
+        /// Scale to measure the percentage or other increase the scaling of the ratio.
+        /// </summary>
+        private double scale;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RatioCounterGauge"/> class.
+        /// </summary>
+        /// <param name="name"> Name of the SumUpCountersGauge.</param>
+        /// <param name="numeratorGauge">The numerator for computing the ratio.</param>
+        /// <param name="denominatorGauge">The denominator for computing the ratio.</param>
+        /// <param name="scale">Scale to measure the percentage or other increase the scaling of the ratio.</param>
```

Comment:
The grammar of this comment doesn't make sense.
But I love that there are comments! :)